### PR TITLE
RavenDB-19967: Notification/warning about tombstone cleaner subscribers disabled

### PR DESCRIPTION
**Pull request by ArieSLV**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-19967/Notification-warning-about-tombstone-cleaner-subscribers-disabled.

### Additional description

The structure of the Notification's details about the presence of tombstone cleanup blocking has been modified:
1. The Source field now stores the name of the Task that, when disabled, leads to the blocking of tombstone cleanup.
2. Two new fields, TaskId and TombstoneDeletionBlockerType, have been added to provide the ability to generate a direct link to the task editing page of the task that, when disabled, results in tombstone cleanup blocking.
3. Additionally, information about the size of the tombstones whose cleanup is blocked has been included in the details of this Notification.

### Type of change

- Bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Non breaking change

### Is it platform specific issue?

- No

### Documentation update

- This change requires a documentation update.

### Testing by Contributor

- Tests have been added that prove the fix is effective or that the feature works.

### Testing by RavenDB QA team

- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- It requires further work in the Studio. Please mark the issue on YouTrack using Studio Required tag.

### CODE REVIEW

File: src/Raven.Server/NotificationCenter/TombstoneNotifications.cs [hunk 1]
```diff
@@ -88,12 +86,55 @@ public DynamicJsonValue ToJson()
...
+    }
+
+    public class BlockingTombstoneDetails
+    {
+        public string Source { get; set; }
+        public TombstoneDeletionBlockerType BlockerType { get; set; }
+        public long BlockerTaskId { get; set; }
+        public string Collection { get; set; }
+        public long NumberOfTombstones { get; set; }
+        public long SizeOfTombstones { get; set; }
```
Comment (ppekrol) [inline src/Raven.Server/NotificationCenter/TombstoneNotifications.cs]: SizeOfTombstonesInBytes

File: src/Raven.Server/Documents/DocumentsStorage.cs [hunk 2]
```diff
@@ -1337,16 +1336,25 @@ public long TombstonesCountForCollection(DocumentsOperationContext context, stri
...
+        }
 
-            return table.NumberOfEntries;
+        public long TombstonesCountForCollection(DocumentsOperationContext context, string collection)
+        {
+            var table = GetTombstoneTableForCollection(context, collection);
+            return table?.NumberOfEntries ?? 0;
+        }
+
+        public long TombstonesSizeForCollection(DocumentsOperationContext context, string collection)
```
Comment (ppekrol) [inline src/Raven.Server/Documents/DocumentsStorage.cs]: TombstonesSizeForCollectionInBytes

File: src/Raven.Server/NotificationCenter/TombstoneNotifications.cs (hunk 1 above)
Comment (ArieSLV) [reply src/Raven.Server/NotificationCenter/TombstoneNotifications.cs]: Done.

File: src/Raven.Server/Documents/DocumentsStorage.cs (hunk 2 above)
Comment (ArieSLV) [reply src/Raven.Server/Documents/DocumentsStorage.cs]: Done.

**Review by grisha-kotler — APPROVED** [review]

**Comment by grisha-kotler** [comment]
The studio doesn't compile

**Pull request merged**